culture/remote/README.md

```diff
@@ -0,0 +1,56 @@
+# Working with our Remote Teammates
+
+At Sparkbox, we have the opportunity to work with many different people in many different locations. Whether it's with our coworkers or our clients, sometimes, we all have to work remote.
+
+## Tips for Non-remote Folks When Working With Remote Colleagues
```

Comment:
Useful tips! I'm glad these things are being documented here. Many of the notes around using Slack and Zoom weren't obvious to me.